data/db/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/UserDataStore.kt

```diff
@@ -89,12 +90,24 @@ abstract class UserDataStore {
         )
     }
 
+    fun language(): Flow<Lang?> {
+        return flowSettings.getStringOrNullFlow(KEY_LANGUAGE).map { it?.let { Lang.valueOf(it) } }
+    }
+
+    suspend fun changeLanguage(language: Lang) {
+        flowSettings.putString(
+            KEY_THEME,
```

Comment:
Probably.. Is this right? 🎨 

```suggestion
            KEY_LANGUAGE,
```